ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -207,7 +207,13 @@ dataman start
 # only start the simulator if not in replay mode, as both control the lockstep time
 if ! replay tryapplyparams
 then
-	simulator start -c $simulator_tcp_port
+  if [ -z "${PX4_SIM_HOST}" ]; then
+    echo "PX4 SIM HOST: localhost"
+    simulator start -c $simulator_tcp_port
+  else
+    echo "PX4 SIM HOST: $PX4_SIM_HOST"
+    simulator start -c $simulator_tcp_port $PX4_SIM_HOST
```

Comment:
Wouldn't it make more sense if the host is passed through a flag as it is done with the ports?

Reply:
I didn't use a flag since currently the flag is only used to specify the protocol. I decided to use the length of the arguments as it was already being used to verify that the port was passed through.